simple-mbed-cloud-client.cpp

```diff
@@ -147,22 +163,11 @@ int SimpleMbedCloudClient::init() {
     palStatus_t status = PAL_SUCCESS;
     status = pal_fsRmFiles(DEFAULT_FIRMWARE_PATH);
     if(status == PAL_SUCCESS) {
-        printf("[SMCC] Firmware storage erased.\n");
+        tr_info("Firmware storage erased");
```

Comment:
Can we get this message enabled by default please?

Reply:
Done.